randovania/games/dread/pickup_database/pickup-database.json

```diff
@@ -268,6 +268,7 @@
             "model_name": "powerup_icemissile",
             "offworld_models": {
                 "am2r": "sItemIceMissilesDread",
+                "planets_zebeth": "spr_ITEM_Ice_Beam",
```

Comment:
I would remove this and thus make it default to the generic offworld model for now.